### PR TITLE
Add goals section

### PR DESCRIPTION
**Pull request by dmose**
as suggested in issue #6.  @Mossop, @joewalker, I'd love to hear your thoughts...

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -13,6 +13,21 @@ For any issues you discover, please file an issue or PR on Github.  If it's
...
 reaching out to me in #nodejs on Slack or [email](mailto:dmosedale@mozilla.com).
 
+### Goals
+Current goals of this policy include:
+
+Make the use of NodeJS/npm packages in the top-level package.json
+package.json well-managed using policies that will result in a set of
+vendored-in packages that:
+
+* are easily auditable and regularly audited for known security issues and
```
Comment (joewalker) [inline index.md]: We should perhaps make it clear that CI isn't the direct goal here, rather to enable another project to add CI (maybe also - regularly auditing, we're enabling that, not actually doing it)?

```suggestion
* are auditable for known security issues and license policy compliance
* can easily be manually audited or made a part of a Continuous Integration process
```

(It looks like Github can't handle multi-line suggestions, some adjustment may be needed, don't worry about attribution)

Comment (dmose) [reply index.md]: Yeah, this makes sense.  Thanks for the suggestion.

File: index.md [hunk 2]
```diff
@@ -13,6 +13,21 @@ For any issues you discover, please file an issue or PR on Github.  If it's
 helpful, you can also give me feedback directly by
 reaching out to me in #nodejs on Slack or [email](mailto:dmosedale@mozilla.com).
 
+### Goals
+Current goals of this policy include:
+
+Make the use of NodeJS/npm packages in the top-level package.json
+package.json well-managed using policies that will result in a set of
```
Comment (Mossop) [inline index.md]: duplicated package.json

**Review by Mossop — APPROVED** [review]

File: index.md (hunk 2 above)
Comment (dmose) [reply index.md]: Good catch; fixed.  Thanks!

**Pull request merged**